### ISSUE TITLE
support Standards item type

### ISSUE THREAD

**Issue by mmoole** [zotero/zotero-bits#52]
First lets recap whats already been on the forums regarding the standards item type:
http://forums.zotero.org/discussion/2914/
btw the link to the IEE manual didn't work for me, but this one works: https://development.standards.ieee.org/myproject/Public/mytools/draft/styleman.pdf see chapter 18.2

Although standards have a lot of specific information, for citing we only need the most important things. 
So lets just start with the most important fields on the top:
- designator for the body org in short form - i.e. ISO/IEC/DIN/ but this can also be a combination like DIN EN ISO.…
- designator for the specific standard - usually a number, i.e. 10646
- number of the part if applicable, i.e. -4 (when citing ISO/IEC 7498-4)  
- year - this is also often followed with a colon directly by the number, i.e. 10646:2003
  well, it could also make sense to include all these four into one field. This might make it much easier!?
- title - the title of the standard
- title of the standards part, if applicable, i.e. Part 4: Management framework
- long form of the body organization/issuing authority - could also be a company name
- pages
- ISBN number if applicable
- language
- issuing workgroup
- short title
- place
- country
- abstract
- URL

maybe:
- ICS class - for example 01.040.31; 31.260 - see for info: http://en.wikipedia.org/wiki/International_Classification_for_Standards 
- DOI
- type or genre: specification, test method, procedure, guide etc…

i hope i didn't miss a lot, so feel free to comment and make more suggestions :-)

**Comment by balrok** [comment]
So what is the item-type one should use? If this bug won't be fixed maybe the documentation can reflect the right item-type.

**Comment by ewa** [comment]
Has anything happened recently on this (beyond Dan Forsberg's file here: http://forsberg.fi/zotero.html)

**Comment by joelotz** [comment]
I agree, has anything happened on this discussion?

**Comment by adam3smith** [comment]
The whole point of a ticket is to allow people to see if there's any progress. There hasn't been.

**Comment by mxa** [comment]
I'd like to see progress here. 5 years later.

**Comment by rmzelle** [comment]
@mxa, it would help us if you and other interested parties could give us links to the relevant sections of popular style guides on the topic of standards, so we can make an inventory of what types of fields would be required to support this kind of item type.

**Comment by jasonzou** [comment]
Here is an example in APA 6th ed. http://libraryguides.vu.edu.au/apa-referencing/patents-and-standards
Standard code/number is formatted differently.

**Comment by ewa** [comment]
Several examples are provided here: http://guides.library.uoit.ca/standards/citing , with references to their sources. 
* IEEE: https://development.standards.ieee.org/myproject/Public/mytools/draft/styleman.pdf#page=35
* ASME: http://libraryguides.missouri.edu/mae/asmecitation

**Comment by ewa** [comment]
In practice, I generally use the techreport BibTeX item type.   So this specification produces the BibTeX (exported from Zotero) below.
![marked up spec title page](https://user-images.githubusercontent.com/288020/33902112-3de86476-df31-11e7-95a0-d5dbffac53ec.png)

```bibtex
@techreport{36785V2V,
    title = {36.785: {{V2V Services}} Based on {{LTE}} Sidelink; {{User Equipment}} ({{UE}}) Radio Transmission and Reception},
    timestamp = {2016-06-15T15:53:26Z},
    number = {36.785},
    institution = {3GPP},
}
```
The main challenges here are:
* Standards almost never seem to have a designated author, and if they do, that's not usually what you want to reference them by.  So that throws off lots of styles which *at best* complain about missing authors.   That leads to throwing off how references are sorted.  One "solution" is to list the issuing group / standards body as the author, but that ends up being redundant if you provide that information anywhere else.
* Standards are so often referred to by their number (or other designator) that you really want it to be the cross-reference key (unless you're just using numbered refs), and it should be the first part of the "title" in the bibliography.   But incorporating it into the "title" field is kind of an ugly hack and gets redundant if you also include in a structured field.
* There are lots of sub-types and extra fields that capture useful information but *may* not always need to be displayed.

If I could have anything I wanted, the reference data would be something like this:
```bibtex
@Specification{36.785V2V,
  title = {{V2V} Services Based on {LTE} Sidelink; User Equipment({UE}) Radio Transmission and Reception},
  title_long = {3rd Generation Partnership Project; Technical Specification Group Radion Access Network; {V2V} Services Based on {LTE} Sidelink; User Equipment({UE}) Radio Transmission and Reception (Release 14)},
  subtype = {TR},
  number = {36.785},
  version = {V14.0.0},    [comment:  Maybe this should be 14.0.0 instead?]
  issuer = {{3GPP}},
  issuer_long = {3rd Generation Partnership Project},
  sub_issuer = {TSG RAN},
  sub_issuer_long = {Technical Specification Group Radio Access Network},
  special:3gpp:date_rev = {2016-10},
  special:3gpp:release = {14}
}
```
A natural sort key (in places where we'd typically use the author) would be `3GPP TR 36.785`, and a reasonable rendering (following the ASME guide) would be

> 2016, 3GPP TR 36.785: Vehicle to Vehicle (V2V) services based on LTE sidelink; User Equipment (UE) radio transmission and reception.

I could probably be convinced that, even though the citation guides don't say anything about it, a version should be considered part of the specification number, giving something like:

> 2016, 3GPP TR 36.785 V14.0.0: Vehicle to Vehicle (V2V) services based on LTE sidelink; User Equipment (UE) radio transmission and reception.

**Comment by ewa** [comment]
I realize those aren't complete answers, but maybe they're helpful @rmzelle.  3GPP makes puts out the weirdest, hardest-to-reference standards I deal with, so if a solution works for them it's off to a good start.

**Comment by WarthogARJ** [comment]
I've been asking for this feature on the Zotero forum since 2006 as well.

I like @ewa format idea.
"Sub-type" is very important, it can be used to refer to a Draft, or to a Recommended Practice as well.

Another comment is that the US standards environment is the exception and not the rule in terms of international practice.

Most countries have in effect STANDARDIZED their standards, and don't have the many 1,000's that the US has. Look at the entire European Community for example, they merged their national standards bodies into one central body: CEN (issues EN standards). It's not just the European Union, but contains other linked countries like Switzerland. Many countries outside the EU import EN standards: this happens in South Africa and Australia for example. CEN has formal links with a lot of the major non-EU standards bodies, like Japan. They only have an **_informal_** relatonship with ANSI (they use this exact term).

And many ISO standards are based on EN standards, in fact CEN and ISO have close ties, understandable since ISO is European based.

The US doesn't really have a central body that issues/controls standards. ANSI is the closest you get, but it doesn't actually draft its own standards.

Anyways, I would suggest that in terms of international coverage, that Zotero ensures its eventual standard type to be a close fit to ISO and CEN standards. That will still fit the major US standards organizations. I think trying to make it fit to the (many 1,000's of) minor US standards issueing bodies is counter-productive: it's better to keep up the pressure on them to merge into a more uniform output.

This has already happened to a large extent with the US military MIL-SPEC's. They decided it was costing too much to maintain the huge number they had, and have decided to cut them back to the few they really need and use civilian specs instead.

**Comment by ewa** [comment]
I looked into adding a "specification" type on my own local branch of Zotero, and concluded that it was a bigger PITA than expected:  The definition of the item <-> itemType <-> field(s) <-> value(s) relation sseems to be spread over a bunch of different files, and adding a new itemType breaks synchronization with the server.

So ... is there a not-super-painful way for a user/developer who isn't a Zotero expert to prototype out an implementation before suggesting it as a change to Zotero proper?  Or is this just one of those "leave it to the experts" things?

**Comment by adam3smith** [comment]
It's fairly involved for a non-expert and it completely breaks your copy of Zotero for sync. What's your interest in prototyping this on a fork? It's not going to make this move any faster -- for that, a simple list of fields and mapping to CSL (and other Zotero fields) with mutual agreement from several folks knowledgeable on referencing standards and links to citation examples (much of which is already present in this thread here). Translating this into database and display changes is indeed a job for the developers, but the real expertise needed here is substantive.

**Comment by ewa** [comment]
I suppose that's true.  My interest in prototyping it is just the usual - I'd like to use it.  If it were easy, I'd set up something for my own use, and maybe learn a few things that might inform what gets implemented for real later.  But it looks to be not so easy.

**Comment by adam3smith** [comment]
Setting this up to show a couple of extra fields and a new item type _is_ fairly easy, but as I say, that breaks a lot of other things, some over which you have no control (like sync), so I wouldn't think it's worth it. It also likely would not work well with updates and not be compatible with the actually implemented solution later on, so if this is intended for actual work, I'd advise against it.

**Comment by charabaruk** [comment]
Something to make note of is that some documents that could fall under this type may be part of different standards series -- for example, RFCs 2119 and 8174 are also (jointly) BCP 14 -- and all these series identifiers may need to be included in a citation for the standards.

Looking at text citations for these two RFCs as provided by the RFC editor website, we see: ([1], [2])

```
Bradner, S., "Key words for use in RFCs to Indicate Requirement Levels", BCP 14, RFC 2119, DOI 10.17487/RFC2119, March 1997, <https://www.rfc-editor.org/info/rfc2119>.
```
```
Leiba, B., "Ambiguity of Uppercase vs Lowercase in RFC 2119 Key Words", BCP 14, RFC 8174, DOI 10.17487/RFC8174, May 2017, <https://www.rfc-editor.org/info/rfc8174>.
```

There also are plenty of standards out there which are shared between standards series published by different institutions without a joint issue sharing the same identifying number in each series; off the top of my head I can think of Office Open XML (ECMA-376, ISO/IEC 29500) and there are certainly others.

[1]: https://www.rfc-editor.org/refs/ref2119.txt
[2]: https://www.rfc-editor.org/refs/ref8174.txt

**Comment by redleafnew** [comment]
hope it finish soon

**Comment by bdarcus** [comment]
Just wanted to try to make sense of this all, and address my earlier concerns. [This link](https://guides.library.uoit.ca/standards/citing) that someone added is, I think, helpful to do that.

Based on that, and what I see in this thread; it's possible we actually don't need any new fields, because we're still dealing with titles, document numbers, URLs, (corporate) authors, etc.

So why the new type for CSL?

Because, as someone here documented, it's common that rendering for these differ from the default "document" type.

My suggestion: let's just accept this, and add a single change to the schema and specification: a new "standard" type.

We can then make clear in the specification the relation to "specification."

Does that make sense?

**Comment by denismaier** [comment]
Yes, absolutely. If we'll realize there's a need for more fields it will be possible to address this in later release.

**Comment by lingr7** [comment]
[kb:style standards [Zotero Documentation]  ](https://www.zotero.org/support/kb/style_standards#commercial_style_guides)

I did n’t find a solution, and I did n’t have the knowledge and ability to provide help on the zotero-wiki, but I think it is often cited that the standard will be in BibTex, which introduces a BibTex solution using python for the 3GPP standard entry mentioned upstairs.
[3gpp-citations · PyPI ](https://pypi.org/project/3gpp-citations/)
[martisak/3gpp-citations: Generate .bib-file for 3GPP specifications ]( https://github.com/martisak/3gpp-citations)

**Comment by WarthogARJ** [comment]
I was in on this quite early, and after reading through the comments, I think it's important to not be too restrrictive on what items you list as a "Standard".

And I think it might be best to call it a "Specification" in a generic sense.
I'd argue that all Standards are Specifications, but not the obverse.

Speaking as an engineer, we use a large range from legally required "specifications"/"standards" to "guidelines" and "recommended procedures".
And depending on your location, what you use might be a legal requirement for you, but is just a guideline in another country.

And standards can be Drafts, for Review, Current, or Withrawn/Replaced.

So my point is that to be really useful, you need to be able to be pretty flexible about what you put into the various fields: it cannot all be dumped into "Extra".

And there are too many standards bodies to have templates for all "Standards"

So forgive me if I'm saying what's not practical, or been said already, but I think the main structure should be:
Title:
Issueing Organisation:
Reference number:
Year Issued: (or valid from, whichever is relevant)
Type: this should be free form, where you can type insomething like "Guideline", "Recommended Practice", "Public Review", "Working Draft", "Specification" etc etc

And then a bunch of fields to try to allow entry of all the other stuff, but I think the first ones are most important.

**Comment by bwiernik** [comment]
The `standard` item type is already implemented in CSL 1.0.2. There is no further need for justification.

**Comment by redleafnew** [comment]
I would like to know which beta version will support standard item type?

**Comment by dstillman** [comment]
@bwiernik: Can you summarize the fields and CSL mappings required?

**Comment by bwiernik** [comment]
Sure 

@rmzelle @adam3smith I'd appreciate your input here. Engineering standards aren't an area I'm that familiar with

**Comment by redleafnew** [comment]
Will the standard item type be supported in zotero recently ?
Thanks.

**Comment by redleafnew** [comment]
Is there any progress for the standard item type?

**Comment by dstillman** [comment]
I can add this as soon as I'm given the final list of fields and CSL mappings.

**Comment by bwiernik** [comment]
@redleafnew What are the fields you'd expect for a Standard item?

@adam3smith @bdarcus Do we know anyone who works with standards that could help with fields?

**Comment by redleafnew** [comment]
I think the fields in JurismM is reasonable: Title, Author, Number, Publisher, Version, Date, Language. Short Titel, URL, Accessed, Archive, Loc. in Archive, Library Catalog, Call Number， Rights, Extra, Data Added, Modified. Besides, Pages and Publisher-place are needed.

**Comment by linjaaho** [comment]
> @adam3smith @bdarcus Do we know anyone who works with standards that could help with fields?

I work with international electrotechnical standardization (IEC), as the convenor of two groups preparing the standards (TC 1 MT 100 and PC 128 WG 2). ISO and EN standards are very familiar too, as well as American SAE-standards. What kind of assistance you do need?

**Comment by linjaaho** [comment]
@dstillman @adam3smith @redleafnew @bwiernik 
When citing EN/ISO/IEC/SAE standards, the following fields are (in my opinion) to be included ([from here](https://docs.citationstyles.org/en/stable/specification.html)) :
- authority (IEC, SAE, IEEE, etc.)
- language
- title
- URL
- edition
- issued
- number
- page

For instance, with IEC: IEC 60364-7-722:2018 ( https://webstore.iec.ch/publication/29958 ):
- authority: IEC
- language: en
- title: Low-voltage electrical installations - Part 7-722: Requirements for special installations or locations - Supplies for electric vehicles
- URL: https://webstore.iec.ch/publication/29958 
- edition: 2.0
- issued 2018-09-21
- number: IEC 60364-7-722:2018
- page: the page(s) to be referenced

These fields are adequate when citing the standards with APA ([1](https://blog.apastyle.org/apastyle/2017/01/how-to-cite-quality-standards-and-guidelines-in-apa-style.html), [2](https://libraryguides.vu.edu.au/apa-referencing/7StandardsPatents)) or [IEEE](https://libraryguides.vu.edu.au/ieeereferencing/standardsandpatents).

**Comment by adam3smith** [comment]
Thanks both -- so going by the lists above, a couple of remaining questions and some comments:
1. do we need `publisher` _and_ `authority` or just one of the two? And if the latter, which is better? (I'd be weakly inclined towards `authority`). In either case, how should this field be called in Zotero?
2. Do we need `publisher-place` as suggested by @redleafnew above? (I don't currently think so, but OK if we do)
3. Do we actually need a `page` field? Specifying a page range cited would be via locator and those are available anyway?
4. We'll definitely want `genre` (I imagine we'd keep this Zotero "type") as described in some of the earlier posts
5. I think `version` makes more sense than `edition` as suggested by @linjaaho 

We'll definitely want DOI -- I'm sure there are already standards with DOI, and it's going to become more common.

So my current proposal would be:

- Standard fields: Title, Author/Contributor, Date, Language. Short Titel, URL, DOI, Accessed, Archive, Loc. in Archive, Library Catalog, Call Number, Rights, Extra, Data Added, Modified
- Issuer (CSL: authority (IEC, SAE, IEEE, etc.))
- Publisher and Place of Publication ([see below](https://github.com/zotero/zotero-bits/issues/52#issuecomment-1207832813))
- Standard number (CSL: number)
- Version
- Type (CSL: genre)

**Comment by redleafnew** [comment]
I think page, publisher-place and publisher is necessary, We are asked to cite them in Information and documentation—Rules for bibliographic references and citations to information resources (GB/T 7714-2015) https://wjk.usst.edu.cn/2020/0523/c10336a220878/page.htm

**Comment by adam3smith** [comment]
Could you give an example of what they would be? I can't get the Chinese to translate well enough & the only example with "standard" in English refers to a book.

**Comment by redleafnew** [comment]
`National Information and Literature Standardization Technical Committee. Bibliography: Part 4: Non book materials: GB/T 792.4-2009 [S]. Beijing: China Standards Press, 2010: 3`
where 
National Information and Literature Standardization Technical Committee--Author
Bibliography: Part 4: Non book materials--Title
GB/T 792.4-2009--Number
[S]--item type
Beijing--Publisher palace
China Standards Press--Publisher
2010--Year
 3--Page

**Comment by adam3smith** [comment]
OK, so that's clear on Publisher & Publisher-place, so let's include those, but for page, this seems like a locator rather than a page range? Recall that the definition of CSL `page` (and the corresponding Zotero `Pages`) is : "Range of pages the item (e.g. a journal article) covers in a container (e.g. a journal issue)"

**Comment by dstillman** [comment]
To clarify, this is where the page locator is set:

https://www.zotero.org/support/word_processor_plugin_usage#customizing_cites

**Comment by redleafnew** [comment]
I think it is pages, in another example it is `2-3`
![image](https://user-images.githubusercontent.com/22135631/183382701-67a85cc9-b5cb-458e-a62c-a541218ad047.png)

**Comment by adam3smith** [comment]
Locators can be ranges as well. `pages` would cover the entire page range of a publication contained in a container publication. That really doesn't appear to be the case here?

**Comment by redleafnew** [comment]
ok. Is there a page field in zotero?

**Comment by adam3smith** [comment]
See dstillman's link above -- a specific cited page or page range wouldn't be in the metadata for an item, it would be added to a citation.

**Comment by redleafnew** [comment]
however the bibliography needs to show the page

**Comment by adam3smith** [comment]
but that's the same with books etc., isn't it? 
That's just a systematic decision in CSL (and Zotero) to not include: adding specific pages to the bibliography in numeric styles is done occasionally, but is conceptually very strange and I wouldn't really know how to support this. 
Certainly we wouldn't want users to create a separate version of an item with a different number in the page field every time they cite a specific page of a work.

**Comment by redleafnew** [comment]
yes. Sometime standards are published like a book in China.

**Comment by redleafnew** [comment]
So I think the pages field could be included.

**Comment by dstillman** [comment]
I think you're misunderstanding this. Zotero already provides a way to cite specific pages in a book. Read the section I linked to above, and try one of the word processor plugins for yourself.

The only actual page-related field book items have is `numPages`. Citing of specific pages is taken care of by the word processor plugins.

**Comment by redleafnew** [comment]
ok, then could the `numPages` be included?

**Comment by adam3smith** [comment]
It'd make more sense, but I really think you're misunderstanding the discussion above: Let me say this again: using specific page numbers in the bibliography of numeric citation styles isn't supported by Zotero or CSL and likely won't be for a long time if ever. That has nothing to do with the standard item type.

Using other fields to hack around this is just going to cause problems, so we certainly wouldn't want to encourage that in any way.

**Comment by redleafnew** [comment]
Got it, many thanks.

**Comment by adam3smith** [comment]
OK, if @bwiernik (and anyone else with an opinion on this) agrees, I think we [have a proposal](https://github.com/zotero/zotero-bits/issues/52#issuecomment-1207766229)

**Comment by linjaaho** [comment]
> Thanks both -- so going by the lists above, a couple of remaining questions and some comments:
> 
> 1. do we need `publisher` _and_ `authority` or just one of the two? And if the latter, which is better? (I'd be weakly inclined towards `authority`). In either case, how should this field be called in Zotero?
> 2. Do we need `publisher-place` as suggested by @redleafnew above? (I don't currently think so, but OK if we do)
> 3. Do we actually need a `page` field? Specifying a page range cited would be via locator and those are available anyway?
> 4. We'll definitely want `genre` (I imagine we'd keep this Zotero "type") as described in some of the earlier posts
> 5. I think `version` makes more sense than `edition` as suggested by @linjaaho

1. I'm also inclined towards authority, but calling it publisher is not a problem too. A good name for it could be Organization, Issuer or Body.
2. For some citation styles, maybe yes. Especially some educational institutions with their personal citation styles are finicky and requiring the place (which is a little weird, who needs the place with ISO or IEC)? But including it would serve users better.
3. I think this is technical issue, as discussed above.
4. No opinion on that.
5. In the world of standardization, it depends on the body, which is used. Usually it is called an edition. In the CEN/CENELEC, IEC and ISO, the word edition is used, but for instance ETSI uses word version.

**Comment by linjaaho** [comment]
To be clear: I agree with your proposal, these are just suggestions.

**Comment by dstillman** [comment]
We have "Issuing Authority" (used for patents and mapped to `authority`) translated already. Do we want to use that, or call it "Issuer"? Fine if we think "Issuer" is more appropriate here.

**Comment by linjaaho** [comment]
> We have "Issuing Authority" (used for patents and mapped to `authority`) translated already. Do we want to use that, or call it "Issuer"? Fine if we think "Issuer" is more appropriate here.

Issuing authority is just fine – the word could be also SDO or publisher but all these are clear in my opinion.

https://en.wikipedia.org/wiki/Standards_organization

**Comment by dstillman** [comment]
How about "Standards Body"? "Standards Organization" is a bit too long for the item pane. ("Standards Body" is also the term I'm most familiar with, for what that's worth.)

"Publisher" is something else, as noted above.

**Comment by dstillman** [comment]
Or we could do just "Organization". The "Standards" in "Standards Body" is maybe a bit redundant in this context, but I'm not sure if just "Organization" is too vague.

**Comment by dstillman** [comment]
Standards Body:

<img width="374" alt="Screen Shot 2022-12-22 at 10 56 47 PM" src="https://user-images.githubusercontent.com/783082/209268169-23cdadeb-b823-4c02-a2fc-1577351219ec.png">

Organization:

<img width="373" alt="Screen Shot 2022-12-22 at 10 57 22 PM" src="https://user-images.githubusercontent.com/783082/209268186-647d7ec8-c845-4d52-8871-82ead3f45c9b.png">

**Comment by WarthogARJ** [comment]
> It's a semantic discussion, but also one about levels of abstraction. I guess you guys are saying they're two words for the same thing. I'm saying they're not, though admittedly not with great certainty.

Sorry, am late to this.
But I was very early to bring the issue up, and I was told nothing could be done at the time due to inability to alter Zotero's framework to any large degree. 

My points:
(1) Item Type: I agree that the overall, umbrella term should be STANDARD.
Not all Standards are Specifications: some are just definitions.
Or Recommended Practices.
Or several other types of standard.

With that in mind, I would consider ISO as the best example to encompass in terms of nomenclature.
And its very name is: "International Organization for Standardization"

So there we can settle two aspects:
"Standard"
"Organisation"

(2) Author
A true Standard does not have an Author(s).
It is a product of a Committee, within a standards body (association).
The actual people who make up the Committee are not pertinent.
However, in Zotero, it would be useful to have the "Organisation" field output in the "Author" field (see below)

(3) Publisher:
Standards are often defined as "Published" but it's misleading to have a field for "Publisher"
"Published" is a STATUS: a milestone in the life of a Standard.
The work of "Publisher" is done by a given Standard's body: they hold any Copyright.
So I don't see a need for anything except "Organisation".

(4) Multiple Organisations:
However, there are often several Standards Organisations linked to a given Standard.
In Europe, it often has three:
- National Standards Body's name (DIN for German, BSI for UK etc)
- CEN (the umbrella EEC body)
- ISO
It often, but not always uses the same number.
But the only difference is a coverpage, belonging to the specific national standards body that issued it (and often translated it to the national language).

So perhaps the field could be "Associated Organisation".
And it's important to have the abbreviation of the Standards Assocation: ISO, CEN, BSI, DIN, NACE, ASME, ASTM etc etc

This is the "Author" of the Standard in terms of how you want it displayed in Zotero.

(5) Committee:
Most standards are drafted by a specific "Committee" (Technical Committee etc).
The field should include room for that since it's a useful piece of information to link related standards together

(6) Status
We need a specific field for STATUS.
It goes from Draft, to Published to Withdrawn.
There are other sub-stages, but these three are the most important ones to cover.

(7) Type
This is a bit misleading I think, unless you want it to be the "Status"
Although you might want to add the closely linked items to Standards such as:
- Working Practice
- Technical Note
- Recommended Practice

(7) Zotero GUI listing tags/fields:
I think it would be very useful to structure "Item Type Standards" to send specific output (that you can select to be displayed in the Listing Field).

For instance, on a standard monitor, you can display the following very usefully of a mixed series of references:
Title (the default)
Creator
Date
Publication

But for Standards, this is not very useful. 

Title: The full standards title is usually too long to be very useful, and most people use a combination of Organisation abreviation plus number instead as the "Title" for a given Standard.
Therefore, I would have the "Title" output field as a concatenation of:
"Number" (c/w Part Number)+ "Year" (Calendar Year only, no month or day) + "Title"
Since "Title" is almost always too long to be displayed, and frequently the distinuguishing words are near the end, "Title" on its own is not very useful

"Creator" Output Field: change this to be Organisation, as an abbreviation.
For example:
ISO
CEN
ASTM
BSI
DIN

"Date" Field is as above, just the Calendar Year: no Date or Month.
It would be misleading to have Zotero output January 1, 2000 if you had "2000" as the "Date" for the Standard
Unless you have "Date" as "Published Date", but most standards users would not be interested in that information, or likely even know it

**Comment by WarthogARJ** [comment]
WRT the definition of a "Standard", and "Type" it can get confusing, especially for American standards since there is no standardisation for American standards (no pun intended). That's because there is no overall ruling national standards body in the USA. ANSI is only advisory and "co-ordinating", unlike the status of national standards bodies elsewhere in the world (for example in Germany it is DIN, in UK it is BSI etc). The USA has a very complex and inconsistent series of associations that issue standards: there's >10,000 of them. Compared to in a given EU Country that has 1 or 2.

(1) The ASTM defines a Standard as:" In ASTM terminology, standards include test methods, definitions, recommended practices, classifications and specifications" (ASTM Charter).

(2) NACE another American-based standards body (National Association Corrosion Engineers, now combined with AMPP) defines a Standard as:
"AMPP  issues  three  classes  of  standards:  Standard  Practices  (SP),  Standard  Test  Methods  (TM),  and  Standard Material Requirements (MR)"

(3) API (American Petroleum Institute) issues the following as "standards":
Standard (Std)
Recommended Practice (RP)
Specification (Spec)
Technical Recommendation (TR)

I recommend calling all the above "Standards", and then distinguishing them by the "Type" field.

**Comment by WarthogARJ** [comment]
Perhaps I missed it, but this issue of how to classify a TECHNICAL standard is already quite exhaustively covered by ISO itself: International Classification for Standards (ICS).

It's like a Dewey-Decimal system for Standards (in fact its website says exactly that).

Wikipedia has a reasonable description of it:
[[]](https://en.wikipedia.org/wiki/International_Classification_for_Standards))(url)

And the current database/listing for the ISO standards following it are at:
[[]](https://www.iso.org/standards-catalogue/browse-by-ics.html)(url)

And Document Center has a Search scheme using it, that includes ISO, ASTM, DIN, BSI, IEC
[[]](https://www.document-center.com/standards/ics/)(url)

ANSI has a search engine using it for American standards:
[[]](https://www.standardsportal.org/usa_en/resources/sdo.aspx)(url)
Also see:
[[]](https://www.standardsportal.org/usa_en/resources/ics_code.aspx)(url)

ICS can be applied to ANY technical standard, and it's actively used by:
ISO
IEEE
ANSI
ASTM
NIST
EN (and related BSI, DIN and other European Standards Organisations)

The goal of ANSI (the US federally funded standards body) is to gradually get all American standards bodies under its wing...which **might** occur before the metric system is adopted...:-}

Therefore, Zotero should have a structured input for ICS classification.
And it would be a useful plug-in to have, where you specify the ICS number and retrieve the Standard info for it.

**Comment by WarthogARJ** [comment]
In fact, ANSI uses the ICS classification itself to classify the various SDO's (Standards Developing Organisations). Look at the tab for "View by Industrial Sector". Each Category is the ICS one.
For example:
Agricultural Machines, Implements, and Equipment - 65.060

Unfortunately due to the embyonic nature of standards standardisation in the USA, not all SDO's are ANSI certified, and very few use the ICS classification on their standards.
So you can use the ICS to go FROM the ANSI site to the SDO, but not the other way.

However, for European standards, the link works both ways.

**Comment by WarthogARJ** [comment]
Hi:
Has anyone read what I wrote here?
I know I wrote a lot, but I tried to do it in an organised manner.
Could I have some quick feedback that it's been at least read?
And if anyone has questions.

I raised the issue of Engineering Standards in 2012:
[[]](https://forums.zotero.org/discussion/26132/item-type-for-engineering-standards#latest)(url)
https://forums.zotero.org/discussion/26132/item-type-for-engineering-standards#latest

And I understood it was on the "To Do" list, and was expected in maybe a year.
I asked again in 2014.
Then in 2018.
Is now late 2022, and is great something can be done now.

No blame, but I made suggestions then, and I think that my above input good, and is well supported.

I work with standards as both a User, and to help draft them.
The UIAA SafeCom issues climbing equipment standards, and this feeds into CEN, which feeds into ISO.

And I'm aware of the differences between US standards and European/ISO ones.
It's pretty easy to address most European ones, because there has been a lot of attention paid to making them consistent: both across 31 European countries, as well as internationally with ISO.
But is not the case in the USA, for better or worse: "it is what it is". 
ANSI is trying to change that, but there's a long way to go.

As I see it, with Zotero, adding an Item Type "Standard" is better than what exists now: which is nothing.
Having "Organisation" is fine too.
But the field "Authors" should not be active if it is for a "Standard".
That is misleading.
Even "Creator" on its own is misleading.
As mentioned below, perhaps substitute "Committtee" for "Authors" if "Standard" is selected.

Generically, ALL Standards follow this route (American, European etc etc):
(1) Some entity requests to another entity that a Standard is generated: this varies, and doesn't need to be in Zotero
Could be a person, or an organisation.
And sometimes the same entity that requests it is the parent organisation that drafts the standard (see below)

(2) Then some entity works on the Standard, and issues a Draft
This Draft should be documented in Zotero, if that's what the item is you want to enter into Zotero
But it's important to note it is a DRAFT.

And we need to document:
Organisation that issued the Draft
Date it was issued
Number of the Draft
And that it is a "Draft"
You can have multiple versions of the Draft, but that is tracked by the Number and/or Date
It would be helpful to have a field for "Committee": if you like, this can be what is filed under "Author"

(3) Issued Standard:
After approved, one could say it has been "Published", but as I said above, not all Standards can be said to have a "Publisher".
Some do, some don't.
Instead, an Issued Standard should have the following:
Organisation that has issued it: this is linked to the NUMBER
Number
Date it is issued (which is the Effective Date usually)

And you often have multiple Organisations that use pretty well the SAME Standard Body
But they usually adjust the Heading/Title to add their Abbreviation to the title
And sometimes adjust the Number too.
And the Language, if translated.

But it will ALWAYS have an associated "Parent" Organisation that did the initial creation of it.
And controls when it is revised, and perhaps withdrawn.

In Europe ( by which I mean the EEC: Europe Economic Community, not just the EU) many standards have 2 if not 3 Organisations linked to them:
CEN: the EEC body that created the Standard
"XYZ": the National Standards Body that uses the CEN standard, and in effect issues it in its own country, and often translates it (and enforces it)
This could by BSI (British Standards Institution), DIN (German) etc etc.
ISO: if the CEN standard has been adopted by ISO. Not all CEN Standards are adopted.

In the USA, you could have two organisations linked to a given Standard:
SDO: Standards Developing Organisation
And sometimes ANSI: the Federal body that is trying to co-ordinate standards across the USA
But unlike the CEN, ANSI does not (tend) to DEVELOP standards: SDO's do that.
Exception are the IEC-related standards.

However, it is not so often (I suspect never: I've not seen it to my recollection) that a "homegrown" standard by most American SDO's would be adopted as-is by ISO,
That is, issued as an International Standard.
The format is not suitable for ISO.
The exceptions are standrds from ASTM, ASME and NIST: they come closest to ISO formats.
But even then, they are changed so much that they are no longer the same item if they are used to help draft an ISO standard.

An example is BS EN ISO/ASTM 52915:2017 Specification for additive manufacturing...
This Standard is IDENTICAL to the ISO/ASTM 52915 (and it says that in the BSI coverpage).

And if you look at the related ASTM listing of it (ASTM ISO/ASTM 52915) the body is SIMILAR but NOT identical to the ISO/ASTM 52915 standard.
There is more than just a coverpage change, and a slight tweak to the title (adding an organisation abbreviatiion in front of the ISO/ASTM 52915).

THEREFORE, my point is, Zotero needs to distinguish the RELATED Organisations with a given standard (and indeed, which one is the PARENT, or perhaps RULING organisation.
And then what organisation has adopted it to in effect re-issue it in its own jurisdiction. 

This is more critical with Standard with an American link.

In Europe, the CEN format is often taken in as-is for an ISO Standard.
Largely because the same people work on the same Committe that issues initially the CEN Stanadard, which is then adopted by ISO.
An example can be seen in BS EN ISO 20312:2012 Petroleum amnd natural gas industries...
This Standard was initially DRAFTED and then ISSUED by CEN (by Tech Committee CEN/TC 12).
That's what the "EN" means in the title.
Then it has been issued in the UK, by the BSI, using the same number, and with a BSO coverpage, but the body is from CEN.
And THEN, it has been adopted "lock stock and barrrel" by ISO, and uses the SAME number.
There is an ISO Committee that confirms it is suitable for ISO as issued: ISO/TC 67).
And a statement that this ISO TC has been prepared "In collaboration" with the CEN TC.

And the EXACT same Standard is issued by ALL the National Bodies of the EEC (31 in total, from Austria thru to UK), but only differing in the first part of the title of the Standard (the National Standards body will have its own initials there), and usually a cover page (and often a translation into the country's own language).

My point is that it's important to identify the "parent" organisation for Standards, as well as the other Organisations that has re-issued it, if that is the case.


Perhaps this all sounds confusing, but if Zotero intends to have some sort of robust consistent classification for Standards, then this is important to understand.

**Comment by adam3smith** [comment]
> Has anyone read what I wrote here?

You do realize when you posted this, yes?

There is indeed a bit too much going on, but for what I gather are the main new issues:

1. **Author/Creator**: we have creator fields currently for all item types, including things like `case` and `statute` where they don't make sense, so I don't think this does much harm here, though I'd also not have a problem with leaving it out.

2. **Status** makes sense to me. I'd add that to the above
3. **Publisher**: is explicitly discussed and addressed as separate from organization in (at least) the Chinese case above, so that's staying in addition to organization/Standards Body. May not exist or be relevant for all cases
4. **Related Organizations/Bodies**: It's not clear to me how that would be incorporated reasonably into the Zotero data model or how it appears in citations (does it ever?). As I understand it, you'd cite one specific version of the standard -- say the Slovenian one -- even if it's understood to be identical in content to a parent CEN/ISO one. The different organization abbreviations are part of the standard number which is how I'd imagine they enter the citation. Doing this via the author field is going to get incredibly messy.

**Comment by dstillman** [comment]
There are examples with authors in this thread. IETF RFCs have authors.

I assume for a committee you'd just use the single-field mode.

**Comment by WarthogARJ** [comment]
OK, thanks for feedback.
And sorry for being impatient.

(1) Author/Creator field:
No offence, but I'd say just because this is incorrect for Case And Statute, that it's a good idea to **add** to the error with a 3rd misleading case.
I would say this being the 3rd time it would be wrong, and I think misleading, helps justify a fix.

Perhaps the best would be to have the "Author" field read "Creator".
For EVERYTHING in Zotero.
Or else have it read "Author/Creator".

Then for Standards, one would enter the Committee that wrote it.

(2) Status: 
 Great.
I think it needs three types: Draft, Active, Withdrawn

(3) Publisher:
No offence, but citing the Chinese case is "tail wags dog".
The bulk (>99.9%) of standards that people use are, in this order, European-based then US-based.
In terms of Users, Population, GDP etc etc.
I think a Zotero Standards entry should aim at being a good match for these types of standards.
See below.

(4) Multiple Bodies:
"It's not clear to me how that would be incorporated reasonably into the Zotero data model or how it appears in citations (does it ever?):
Yes: it occurs for most European/American technical standards.
And Yes, I don't think including it in Authors would be ideal: rather use the Committee for "Author" as the "Creator".

One prime case of being useful is for the bulk of American standards.
That are issued by an SDO, and without any reference to ANSI.
But however are indeed acredited by ANSI.

A Zotero user might not decide to enter anything for any other Body, but the option should be there. 
Once the Zotero Standard starts going, hopefully a plugin will be written to add the relevant information.

Perhaps this is not obvious to people with a non-technical foundation, but standards by their nature are....standardised.
Compare them to research published in Journals.
There are thousands of Journals, with no standardised format.
Whereas for the mainstream engineering technical standards (ISO, EN, ASTM, ASME, IEC) there is ONE format per body.

**Comment by WarthogARJ** [comment]
> There are examples with authors in this thread. IETF RFCs have authors.
Authors:
I would view Standards from IETF to be a very (very) small case out of the overwhelmingly larger number of Standards issued by the mainstream engineering standards organisations: ISO, ASTM, ASME, EN, IEC etc etc.

I would argue that by their very nature, that Standards do not have an "Author".
It's close to anethema for a mainstream engineering Standard to have an "Author", if you look at how "Author" is defined by various works.
If we use the Committee of a given Standard as the "Author", then:
(1) Copyright: none, resides with issueing Organisation
(2) Individual recognition (i.e. to an individual human being): none (see below)
(3) Writer: often (usually) does NOT actually write ANYTHING in the published work, the standard (see below)

A mainstream standard (from ISO, EN, ASTM, IEC etc) has a COMMITTEE comprised of technical matter experts who contribute their expertise, and there is often a 2nd group of professionals from the supporting Standards (parent) Organisation who take that and actually draft (write) the Standard.
For most CEN standards, the supporting body is the German DIN.
For some, like ISO oilfield related standards, it is the French AFNOR.

Are either of these an "Author" in the same sense as they are for other references?
I'd say No.
There are not, and **cannot** be a specific individual.
The Committee is usually comprised of representatives of _stakeholders_.
Sure, sometimes individual experts are on a Committtee, but at the discretion of the Committee Secretary.
And then you'd class them as representatives of the existing subject knowledge: replaceable to other experts

I would argue that compared to other references that Zotero covers, that "Author" for a Standard is split into two parts: 
(1) the actual intellectual input (from the members of the Committee)
(2) the organisation and output of it in the required format (by paid technical writers of the parent body)

 Is closer to someone who hires a ghost writer to write something.
They are both required to fulfill the author duty.

**Comment by adam3smith** [comment]
Sorry, but this is just not a helpful way to engage. Your expertise is welcome, but only if you can contribute it with an open mind and some humility & respect to the expertise and interest of others. Otherwise, I'd suggest writing your own app.

**Comment by mmoole** [comment]
I think there are a lot of perspectives being presented here which may lead to misunderstandings (I love this 🥳), examples being: 
* end users are directly writing to developers, who then could often interpret things being mentioned as requirements (I would always fall for this);
* trying to reflect 'all' use cases in the digital model in Zotero - but we need a trade off somewhere between enough details and not too rough to be unusable;
* subject-matter experts in this or that field chiming in, being understood in not their expressed/intended ways, leading to varying motivation on all sides...;
* etc...

I would add some snippets to these aspects:
* status/state: although I would agree on the status of 'draft', the other states of being 'active'/'effective' or 'withdrawn' tend to not be part of the published instance of the standard. I.e. there is a draft version (or multiple) that's being published as draft, but the then released standard is published only once. It doesn't get republished with another state/status, unlike the draft. This means the state of the released standard changes over time. One mostly would need to look it up at an authoritative body's website. 
If we would be even more complete, I would also suggest 'superseded' as state (as had been mentioned above ( https://github.com/zotero/zotero-bits/issues/52#issuecomment-11839920_ ). 
But in the end, I am not so certain it would make sense to include this at the first CSL release containing standards. Maybe that is an aspect that will develop further once standards are included.

* author/creator: as Zotero already has many kinds of creators for various item types, I don't think we need more here. From an end user point of view it may be nice to see it labeled a bit different from for example at the book type. Here I would also say that once standards are in CSL and Zotero, it may turn out to be useful to have this and that, but those may be different things or made a bit differently than we see things now.

But finally I have to say as an end user I am very happy to use Zotero, and look forward at what the development team will do towards standards. We can contribute input from various points of views here (and this may be quite inconsistent and contain contradictions), but it's up to the real developers to blend this in a fully specified, impeccable CSL schema that will work with Zotero and all CSL using apps out there 🤩.

**Comment by fgnievinski** [comment]
Here are some examples of citations to standards (in APA Style, 7th Ed.):

https://apastyle.apa.org/style-grammar-guidelines/references/examples/iso-standard-references

I don't see anything super specific or incompatible in the usage of authors field.

FGN

**Comment by WarthogARJ** [comment]
> I think there are a lot of perspectives being presented here which may lead to misunderstandings (I love this 🥳), examples being:
> 
> * end users are directly writing to developers, who then could often interpret things being mentioned as requirements (I would always fall for this);
> * trying to reflect 'all' use cases in the digital model in Zotero - but we need a trade off somewhere between enough details and not too rough to be unusable;
> * subject-matter experts in this or that field chiming in, being understood in not their expressed/intended ways, leading to varying motivation on all sides...;
> * etc...
> 
> I would add some snippets to these aspects:
> 
> * status/state: although I would agree on the status of 'draft', the other states of being 'active'/'effective' or 'withdrawn' tend to not be part of the published instance of the standard. I.e. there is a draft version (or multiple) that's being published as draft, but the then released standard is published only once. It doesn't get republished with another state/status, unlike the draft. This means the state of the released standard changes over time. One mostly would need to look it up at an authoritative body's website.
>   If we would be even more complete, I would also suggest 'superseded' as state (as had been mentioned above ( [support Standards item type #52 (comment)](https://github.com/zotero/zotero-bits/issues/52#issuecomment-11839920)_ ).
>   But in the end, I am not so certain it would make sense to include this at the first CSL release containing standards. Maybe that is an aspect that will develop further once standards are included.
> * author/creator: as Zotero already has many kinds of creators for various item types, I don't think we need more here. From an end user point of view it may be nice to see it labeled a bit different from for example at the book type. Here I would also say that once standards are in CSL and Zotero, it may turn out to be useful to have this and that, but those may be different things or made a bit differently than we see things now.
> 
> But finally I have to say as an end user I am very happy to use Zotero, and look forward at what the development team will do towards standards. We can contribute input from various points of views here (and this may be quite inconsistent and contain contradictions), but it's up to the real developers to blend this in a fully specified, impeccable CSL schema that will work with Zotero and all CSL using apps out there 🤩.

(1) End Users writing to Developers:
Point 1: maybe you miss the point, but input has been, and SHOULD be, requested
Point 2: the "Developers" admitted/stated that they NEEDED input, because they don't have specific expertise in Standards
I'm not sure if you do, but if so, you should state SPECIFIC input to Standards, and not just generalised comments

(2) WRT this specific comment:
"But finally I have to say as an end user I am very happy to use Zotero, and look forward at what the development team will do towards standards. We can contribute input from various points of views here (and this may be quite inconsistent and contain contradictions), but it's up to the real developers to blend this in a fully specified, impeccable CSL schema that will work with Zotero and all CSL using apps out there"
I don't see any constructive input here, just trying to throttle other's input.
"Real Developers" iomplies there are "NOT REAL" Developers.
The idea behind Zotero is it's non-profit, and to some extent, open source in that one can write plugins etc for it.
If you want "Real" Developpers to do everything, then perhaps have a look at Endnote.

**Comment by WarthogARJ** [comment]
> Sorry, but this is just not a helpful way to engage. Your expertise is welcome, but only if you can contribute it with an open mind and some humility & respect to the expertise and interest of others. Otherwise, I'd suggest writing your own app.

I've given my input, and support for it.
I don't see that it's not "helpful".
Facts are facts, no need for any emotive spin.

What's "humilty" got to do with it?
Zotero is supposed to be open to suggestions from Users.
And especially where expert input can be used.
I've given lots of examples and references.

AdamSmith is a "Collaborator", and not a Developer or Maintainer of Zotero.
Or as I understand it, a member.

You accuse someone of not being "humble"....???
But who has raised this issue over 10 years ago, and checked back in about it several times, and now has made what are intended to be useful comments.

As far as "writing my own app", and "Developers" etc, who is actually writing code for Zotero in this discussion?
Have they made any comments in this thread?
Zotero is not open source.
It's supported by a non-profit, and they get tax deductions for this.

And it relies on its use by public instututions like universities.
Places that place value on free speech.
And don't try to stiffle input.

**Comment by WarthogARJ** [comment]
> Here are some examples of citations to standards (in APA Style, 7th Ed.):
> 
> https://apastyle.apa.org/style-grammar-guidelines/references/examples/iso-standard-references
> 
> I don't see anything super specific or incompatible in the usage of authors field.
> 
> FGN

Sure: I don't see any issue with how these are formatted.
But it's done manually: not via Zotero.
And this is exactly my point: I don't see how the proposed formats for Standard can provide this specific output.

If the new Zotero Standard can provide it is this manner: great.

**Comment by adam3smith** [comment]
dstillman is Zotero's lead developer. I co-maintain the Citation Style Language, which is responsible for the automated citations produced by Zotero, i.e. directly relevant code for this issue. In fact, the majority of the people posting here have directly contributed to Zotero and/or CSL. 

[humility](https://www.vox.com/science-and-health/2019/1/4/17989224/intellectual-humility-explained-psychology-replication) is relevant because you can't contribute effectively to a collaborative project if you're not willing to listen to and appreciate what other contributors have to say and that you might on occasion be wrong -- see e.g. your dismissal of Internet standards, Chinese requirements for citing standards, as well as the expertise of developers in how to _implement_ specific metadata and citation requirements in software they've been working on for over 15 years.

**Comment by dstillman** [comment]
OK, here's what we have:

<img width="373" alt="standard" src="https://user-images.githubusercontent.com/783082/210191669-88fdb1d7-034c-4062-a53c-96035f794318.png">

(And Abstract will likely move in a future redesign, putting Organization immediately below Author.)

**Comment by adam3smith** [comment]
Looks good to me. The open questions I have from looking again at ISO metadata as well as the discussion & additional detail provided above. For context here's what ISO seems to consider the key metadata for their standards (in addition to number & title)
![image](https://user-images.githubusercontent.com/128986/210192716-be73425b-2979-4171-9c95-f4be15e8cff8.png)
(from [this standard](https://www.iso.org/standard/81870.html))

Based on this:
1. **(Technical) Committee:** I don't think I've ever seen this cited, but if we do want to capture that information (and both given the inclusion by ISO and the above referent to it it does seem relevant info), I think it'd be weird to add it as a single field contributor(?) (and definitely shouldn't be author where we'd have to work around it in every citation style). We do have the (slightly different but related) `committee` for `hearing`, mapped to CSL `section`. No strong opinion on whether this makes sense to include.
2. **Number of pages:** ISO provides this and might be nice to capture where it exists since we have good standard Zotero&CSL support? I'd be inclined towards adding this.
3. **ICS** -- I think the closes analog we have for this are LCSH (also hierarchical, standardized classifiers) so I think automatic tags are probably the way to go here to include those, i.e. nothing to do on that.

I think that's it -- I'm pretty sure this covers all citation examples we have above and a good bit more for eventualities & future developments (and, as for all complex types, obviously doesn't capture the full complexity of available Metadata).

**Comment by dstillman** [comment]
OK, thanks. I'm fine adding `committee` below `organization` and adding `numPages`. And "Edition" is "Version" (mapped to CSL `version`) in this case, yes?

**Comment by adam3smith** [comment]
> "Edition" is "Version" (mapped to CSL version) in this case, yes?

Yes, exactly

**Comment by KingCZE** [comment]
Would you be able to show an example on e.g. 
ČSN EN 1992-1-2:2006 (73 1201) (Or even funnier ČSN P ENV 1992-1-2:1998 (73 1201)); 
BS EN ISO 3834-2:2021 - TC; 
NZS 4541-2020; 
AS/NZS 4187:2014 A2?

I have my doubts about how universal it really is, and some of the fields are not really clear to me. 


Thank you.

**Comment by adam3smith** [comment]
Title: Quality requirements for fusion welding of metallic materials - Comprehensive quality requirements
Number: BS EN ISO 3834-2:2021 - TC
Date: 2021-05-31
Organization: BSI
ISBN: 978 0 539 17816 6

And optionally
Status: Current
Committee: WEE/36
Publisher: British Standards Institution
URL: https://knowledge.bsigroup.com/products/quality-requirements-for-fusion-welding-of-metallic-materials-comprehensive-quality-requirements-1

The others follow the same logic

**Comment by KingCZE** [comment]
> Title: Quality requirements for fusion welding of metallic materials - Comprehensive quality requirements Number: BS EN ISO 3834-2:2021 - TC Date: 2021-05-31 Organization: BSI ISBN: 978 0 539 17816 6
> 
> And optionally Status: Current Committee: WEE/36 Publisher: British Standards Institution URL: https://knowledge.bsigroup.com/products/quality-requirements-for-fusion-welding-of-metallic-materials-comprehensive-quality-requirements-1
> 
> The others follow the same logic

So nothing like dividing "Part 2", subpart name, preliminary code (ENV), catalogue / national number (73 1201),... ?

**Comment by adam3smith** [comment]
Correct. There's no way we're going to disassemble the standard numbers. I understand the individual parts have meaning, but I don't think it makes any sense to store them as individual fields in the context of an app like Zotero.

**Comment by iAnyKey** [comment]
Does anybody working on a standard template right now?

**Comment by adam3smith** [comment]
There's no work left afaict. It'll land with the next batch of new item types/fields

**Comment by IndefiniteBen** [comment]
Is that planned for a specific time, or just when everything is ready?

**Comment by iAnyKey** [comment]
@dstillman may we ask to release these changes?

**Comment by stanlrt** [comment]
ISO reports are still added as web pages...

**Comment by dstillman** [comment]
@stanlrt: That's separate from the existence of a Standards item type. If something isn't being saved correctly, please report it — with example URL — in the Zotero Forums.

**Comment by yasirroni** [comment]
Since this is supported, can anyone explain what is the supposed BibTeX entry types for importing and exporting this?

**Comment by bwiernik** [comment]
There isn't an out of the box or conventional BibTeX or BibLaTeX item type for standards. Some styles define a custom type, but this is not consistent 


I don't recommend using BibTeX as a format for citation data transfer. It's not very complete in its coverage of citation items, and it relies heavily on style-specific data entry, so it is hard to use as a general data transfer format. 

I recommend CSL-JSON as the most complete citation data transfer format

**Comment by adam3smith** [comment]
Biblatex has it listed as a non-standard (ha!) item type though, so we could add there

**Comment by bwiernik** [comment]
Zotero currently exports to misc for both BibTeX and BibLaTeX, would probably be good to update translator
